docs/source/miniconda_hashes.rst

```diff
@@ -7,6 +7,40 @@ Miniconda hash information
+Miniconda3-py37_23.1.0-1-MacOSX-x86_64.sh     53.3 MiB   2023-02-07 21:27:23  ``bdfb2f01c0a3917bf258daffc65b69bfe07e29753be624aaf9cbda5ba02f43f4``
+Miniconda3-py37_23.1.0-1-Windows-x86_64.exe    50.6 MiB   2023-02-07 21:27:23  ``2319e6ab37215daf08f47b0da35a53f6a648121029113ae2ba53917d777b84bd``
+Miniconda3-py38_23.1.0-1-Linux-aarch64.sh     48.2 MiB   2023-02-07 21:27:23  ``10ea91cc579a64a3a88727119ac3f55839562f55118458b82824b544bc74f90d``
+Miniconda3-py38_23.1.0-1-Linux-ppc64le.sh     48.5 MiB   2023-02-07 21:27:23  ``d89faee2d839c7e8a2c96f3ca60295c08e837c2f134f6bb9e9e21b707babedc2``
+Miniconda3-py38_23.1.0-1-Linux-s390x.sh       61.5 MiB   2023-02-07 21:27:23  ``3d1e06eddaef0976530c54ed7dda80df62705c16513634e58f7d1c4567227b9e``
+Miniconda3-py38_23.1.0-1-Linux-x86_64.sh      65.4 MiB   2023-02-07 21:27:23  ``640b7dceee6fad10cb7e7b54667b2945c4d6f57625d062b2b0952b7f3a908ab7``
+Miniconda3-py38_23.1.0-1-MacOSX-arm64.pkg     42.6 MiB   2023-02-07 21:27:23  ``975d6daa8afd01459b99b924703494a23519ed113bac5ba7f7db355904f37b22``
+Miniconda3-py38_23.1.0-1-MacOSX-arm64.sh      42.9 MiB   2023-02-07 21:27:23  ``8dfab7797151a31b16c174da9a5bc09529d5859f21e77f0655ea9b18209cc926``
+Miniconda3-py38_23.1.0-1-MacOSX-x86_64.pkg    43.1 MiB   2023-02-07 21:27:23  ``beed5074ac12b9ef2820f03a3a0efe910cdd545af8fe0aad1d9c190173150f88``
+Miniconda3-py38_23.1.0-1-MacOSX-x86_64.sh     43.3 MiB   2023-02-07 21:27:23  ``5d789cda38b23245ffed6b88c60b7479d984bbf20e3b70d66cd150f04a9c25c5``
+Miniconda3-py38_23.1.0-1-Windows-x86_64.exe    52.4 MiB   2023-02-07 21:27:23  ``4178df2a15284fd07b10c5fad592e5c15e6be5bfc37ee90d8e02bbde7792f6f9``
+Miniconda3-py39_23.1.0-1-Linux-aarch64.sh     60.5 MiB   2023-02-07 21:27:23  ``5e67416a574c49e19dc21d5b9ed586400863a685bc4e34b4d933ea8c7c1ed2da``
+Miniconda3-py39_23.1.0-1-Linux-ppc64le.sh     60.6 MiB   2023-02-07 21:27:23  ``cf5d7cad2b0eb260903b3661ee3fa822eecb25cf3c9b14bc9de10d72963d3d5a``
+Miniconda3-py39_23.1.0-1-Linux-s390x.sh       62.8 MiB   2023-02-07 21:27:23  ``5159322f15d9e2b22b3cf90fe88b336d84f62189178c872a9288a339d86f5d20``
+Miniconda3-py39_23.1.0-1-Linux-x86_64.sh      66.7 MiB   2023-02-07 21:27:23  ``5dc619babc1d19d6688617966251a38d245cb93d69066ccde9a013e1ebb5bf18``
+Miniconda3-py39_23.1.0-1-MacOSX-arm64.pkg     42.7 MiB   2023-02-07 21:27:23  ``b09fa8474db00127701a670886e8608da6e00c4be97d93f5dd57bbd497cdb92a``
+Miniconda3-py39_23.1.0-1-MacOSX-arm64.sh      43.0 MiB   2023-02-07 21:27:23  ``a7133a703e41ea0b1738196fb03f72b22250327adea94521c9dd6100c304dc63``
+Miniconda3-py39_23.1.0-1-MacOSX-x86_64.pkg    43.1 MiB   2023-02-07 21:27:23  ``878c7939731a712ba3dccfccf8df3c0ac8e5a7d7486b43bfc9e422907ecf8ca5``
+Miniconda3-py39_23.1.0-1-MacOSX-x86_64.sh     43.3 MiB   2023-02-07 21:27:23  ``d78eaac94f85bacbc704f629bdfbc2cd42a72dc3a4fd383a3bfc80997495320e``
+Miniconda3-py39_23.1.0-1-Windows-x86_64.exe    52.9 MiB   2023-02-07 21:27:23  ``a2e7ec485e5412673fad31e6a5a79f9de73792e7c966764f92eabf25ec37557f``
```

Comment:
This page didn't render before, see https://docs.conda.io/en/latest/miniconda_hashes.html 

Seems to be still broken https://continuumio-conda-docs--838.com.readthedocs.build/en/838/miniconda_hashes.html

Reply:
created https://github.com/conda/conda-docs/pull/839

Reply:
So currently I'm seeing py310, then py37, py38, and py39 in this table. Any way to order them descending properly? 310, 39, 38, and 37?